docker/pool/builder/lib/builder/docker.rb

```diff
@@ -4,17 +4,21 @@
 module Builder
   module Docker
     CONTAINER_PORT = 80
+    ::Docker.options[:read_timeout] = 15 * 60 # 15 minutes
 
+    attr_accessor :logger
+    module_function :logger, :logger=
```

Comment:
`attr_accessor :logger` is equivalent to define methods below,

```
 def logger
    @logger
  end

  def logger=(new_bar)
    @logger = new_bar
  end
```

http://stackoverflow.com/a/6852781